tcms/rpc/api/testexecution.py

```diff
@@ -217,8 +217,14 @@ def add_link(values, update_tracker=False, **kwargs):
 
             Always 'link' with IT instance if URL is from Kiwi TCMS own bug tracker!
     """
-    link, _ = LinkReference.objects.get_or_create(**values)
-    response = model_to_dict(link)
+    form = AddExecutionLinkForm(values)
+    form.populate(execution_id=values['execution_id'])
```

Comment:
Fests are failing b/c switching to a model form changes the names of the fields representing foreign keys. There is no `execution_id` anymore, it becomes `execution`.

Reply:
Updated in the second commit.